setup-enzyme.js

```diff
@@ -2,3 +2,7 @@ import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 
 configure({ adapter: new Adapter() });
+
+console.error = (message) => { // eslint-disable-line
+  throw new Error(message);
```

Comment:
Does this make all errors thrown in enzyme tests actually log? What's the goal?

Reply:
Jasmine does _not_ fail when `console.error` is called since it is not an actual _thrown_ error. This just makes sure we do not have any `console.error`s in our tests.